src/components/cluster.js

```diff
@@ -83,6 +83,10 @@ export default {
     eventsBinder(this, this.$clusterObject, events)
   },
 
+  updated () {
+    this.$clusterObject.repaint()
+  },
+
```

Comment:
If you see the subsequent lines (lines 90-99) there is already an attempt to try to fix the performance issue.

Lines 93-95 explicitly removes the reference to `$clusterObject` so that the children will not end up calling `removeMarker`.

Do you know why this implementation isn't sufficient? Are your marker objects direct children of the cluster object?

Reply:
When I remove a large amount of markers but leave the cluster in place, the cluster is never destroyed, so the hook never get triggered. Besides, when a clustered marker is removed I suppose you would want to remove it from the cluster as well don't you?
Apart from that, my experience so far is that is I've gone from an unresponsive browser for ~30 seconds to about a second when removing ~1600 markers. (Works on my machine! :)